doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -90,8 +90,8 @@ not set, they look for the commands in the executable path.
 The ``$COQ_COLORS`` environment variable can be used to specify the set
 of colors used by ``coqtop`` to highlight its output. It uses the same
 syntax as the ``$LS_COLORS`` variable from GNU’s ls, that is, a colon-separated
-list of assignments of the form ``name=``:n:``{*; attr}`` where
-``name`` is the name of the corresponding highlight tag and each ``attrᵢ`` is an
+list of assignments of the form :n:`name={*; attr}` where
+``name`` is the name of the corresponding highlight tag and each ``attr`` is an
```

Comment:
Should these be `@name` and `@attr`? (Possibly not, since we don't want them to link to the corresponding grammar entries, but `@` is useful to indicate placeholders.

Reply:
Hum, this dual use of `@` is interesting. Linking to grammar token vs indicate placeholders. But on the other hand, in most cases you want placeholders to have an associated grammar because you want to know what you can replace it with...